.update-copyright.conf

```diff
@@ -0,0 +1,17 @@
+[project]
+name: lesson-template
+vcs: Git
+
+[files]
+authors: yes
+files: no
+
+[copyright]
+short: {project} comes with ABSOLUTELY NO WARRANTY and is licensed under the MIT and CC BY 4.0 licenses.
+long: This file is part of {project}.
+
+  Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
```

Comment:
On Sun, Mar 15, 2015 at 07:55:04PM -0700, Greg Wilson wrote:
> I prefer "See LICENSE.md for copyright and license information." --
> single unambiguous source of the information.

But it doesn't survive a file being copied out of the repository that
it was developed in.  E.g. swcarpentry/amy copied check.py from
swcarpentry/workshop-template.  They have the same licenses in this
case, but someone with a primarily BSD-licensed project might do the
same thing, and then the reference to the local BSD LICENSE.md
wouldn't be right.